i18n/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 from . import config
 
-__version__ = '0.9.2'
+__version__ = '0.10.0'
```

Comment:
This is a backward incompatible change.

```suggestion
__version__ = '1.0.0'
```

Reply:
Done